setup.sh

```diff
@@ -405,7 +405,21 @@ log "Pushing initial cluster configuration"
 git add config.yaml footloose.yaml machines.yaml flux.yaml wks-controller.yaml
 
 git diff-index --quiet HEAD || git commit -m "Initial cluster configuration"
-git push
+
+ORIGN=$(git config --get remote.origin.url)
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+echo 'Working on Repository: '$ORIGN
+echo 'On Branch: '$BRANCH
+
+read -p "Are you sure you want to push to \"$BRANCH\" ? (y/n): " -n 1 -r
```

Comment:
Typically when prompting a user, you uppercase the default.  Please upper case the N

Reply:
I did not know about this naming convention where you have to uppercase the user's default option, thank you.